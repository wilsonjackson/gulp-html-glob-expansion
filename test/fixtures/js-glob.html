<html>
<head>
	<title>You Always Look Fine</title>

	<script src="/assets/*.js"></script>
</head>
</html>
