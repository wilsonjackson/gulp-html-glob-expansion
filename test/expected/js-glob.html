<html>
<head>
	<title>You Always Look Fine</title>

	<script src="/assets/file.js"></script>
	<script src="/assets/otherfile.js"></script>
</head>
</html>
